release.nix

```diff
@@ -15,6 +15,20 @@ let
 in
 
 rec {
+  disciplina-haddock = with lib;
+    let
+      drvs = filterAttrs (const (hasAttr "doc")) project;
+      docs = mapAttrs (const (getOutput "doc")) drvs;
+      globs = map (doc: "${doc}/share/doc/*") (attrValues docs);
+    in
+    runCommand "disciplina-haddock" {} ''
```

Comment:
[pkgs.linkFarm](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/trivial-builders.nix#L146) "disciplina-haddock" ?

Reply:
Current approach looks much better in Buildkite artifacts output.